src/components/Carousel/utils.ts

```diff
@@ -46,11 +46,14 @@ export const getPosition = (index: number, props: CarouselProps): number => {
  * @param forceReflow
  */
 export const setPosition = (position: number, axis: 'horizontal' | 'vertical'): React.CSSProperties => {
-    const style = {};
-    ['WebkitTransform', 'MozTransform', 'MsTransform', 'OTransform', 'transform', 'msTransform'].forEach((prop) => {
-        // @ts-ignore
-        style[prop] = CSSTranslate(position, '%', axis);
-    });
-
-    return style;
+    const cssTranslated = CSSTranslate(position, '%', axis);
```

Comment:
Removed the @ts-ignore and refactored the code a bit. I guess it's a bit of personal preferences which version you prefer.